store/index.js

```diff
@@ -26,24 +27,25 @@ export const plugins = [
   Steve({ namespace: 'management', baseUrl: '/v1' }),
   Steve({ namespace: 'cluster', baseUrl: '' }), // URL dynamically set for the selected cluster
   Steve({ namespace: 'rancher', baseUrl: '/v3' }),
+  Steve({ namespace: 'virtual', baseUrl: '' }),
```

Comment:
Was there a technical reason for creating a specific store for harvester? From what i could remember a lot fo the `if`s could be resolved via looking at the product instead. All the data comes from the steve/v1 api and there's a currentCluster that's the harvester one

Reply:
I think this is mentioned somewhere else, but if the store remains then the name should be updated to something more harvester specific

Reply:
The harvester's data comes from `step/v1/harvester/`, and we may also use the upstream cluster's data `steve/v1/ management. Cattle. *` when entering the harvester cluster, so we created a new store.

Reply:
I changed the virtual store to harvester store

Reply:
Done